docs/metrics-guide/averaging-methods.md

```diff
@@ -0,0 +1,118 @@
+---
+subtitle: Macro, Micro, Weighted
+---
+
+# Averaging Methods
+
+For multiclass workflows like classification or object detection, metrics such as precision, recall, and F1-score are
+computed **per class**. To compute a single value that represents model performance across all classes, these per-class
+scores need to be aggregated. There are a few different averaging methods for doing this, most notably:
+
+- [**macro**](#macro-average): unweighted mean of all per-class scores
+- [**micro**](#micro-average): global average of per-sample TP, FP, FN scores
+- [**weighted**](#weighted-average): mean of all per-class scores, weighted by sample sizes for each class
+
+## Example: Multiclass Classification
+
+Let’s consider the following multiclass classification metrics, computed across a total of 10 samples:
+
+| Class | # Samples | <span title="# True Positives">TP</span> | <span title="# False Positives">FP</span> | <span title="# False Negatives">FN</span> | Precision | Recall | F1-score |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| `Airplane` | 3 | 2 | 1 | 1 | 0.67 | 0.67 | 0.67 |
+| `Boat` | 1 | 1 | 3 | 0 | 0.25 | 1.0 | 0.4 |
+| `Car` | 6 | 3 | 0 | 3 | 1.0 | 0.5 | 0.67 |
+| Total | 10 | 6 | 4 | 4 | - | - | - |
+
+
+#### Macro Average
+
+**Macro average** is perhaps the most straightforward among the numerous options and is computed by taking an
+**unweighted** mean of all the per-class scores:
+
+$$
+\begin{align}
+\text{F1}_\text{macro} &= \frac{\text{F1}_\texttt{Airplane} + \text{F1}_\texttt{Boat} + \text{F1}_\texttt{Car}}{3} \\[1em]
+&= \frac{0.67 + 0.4 + 0.67}{3} \\[1em]
+&= 0.58
+\end{align}
+$$
+
+#### Micro Average
+
+In contrast to macro, **micro average** computes a **global** average by counting the sums of true positive (TP), false
+negative (FN) and false positive (FP):
+
+$$
+\begin{align}
+\text{F1}_\text{micro} &= \text{TODO} \\[1em]
```

Comment:
I have an idea for this.
How about if we start with `micro precision` and `micro recall` and compute `f1` after with precision and recall values so it should be 2 * 0.6 * 0.6 / (0.6 + 0.6) = 0.6.  I think this F1 formula is what readers would be more familiar with.

Reply:
I like this idea better than what's up currently. Updating.